### DISCUSSION TITLE
Trouble configuring SecurityOnion Latest on Proxmox VE / KVM

### DISCUSSION THREAD

**Discussion by tanc7** [Security-Onion-Solutions/securityonion#6153] [Unsupported Versions] [answered]
I just installed SecurityOnion on Proxmox KVM and added two virtual NICs. As a response to this error.

![Screenshot (342)](https://user-images.githubusercontent.com/26513830/140825337-019b695e-f75d-4b63-bbd4-8edfeee10cd1.png)

After adding the second Virtual NIC it appears that Proxmox/KVM has assigned a automatically assigned a IP address that is not the same as what I entered in the setup prompt.

![Screenshot (344)](https://user-images.githubusercontent.com/26513830/140825080-7cd5df10-2160-4e3c-b5e0-6e8c617a5712.png)

Anyone have any experience in configuring SecurityOnion on Proxmox/KVM? Like a write-up guide or blog post? I just want to do the steps right. But I believe I simply disable DHCP for this specific VM on Proxmox, and manually configure them in the SO prompt.

**Comment by Sahale** [comment, marked as answer]
Hello!
If you use want to use DHCP, make sure that you reserved certain IP addresses for SO management interface. If management gets another IP, it can cause problems. Monitor interface doesn't need to have assigned IP at all, so you can safely disable DHCP for this interface.
I'm using SO on Proxmox. But I installed SO over CentOS.
If you want to monitor traffic you have to configure monitor interface on Proxmox in a certain way. Configuration depends on your network and requirements, in my case I monitor traffic from my router via span interface. For that case I enabled promiscuous mode for interface and it was enough, if I remember all correctly. I think it's better to read Proxmox docs or [forums](https://forum.proxmox.com/threads/deploying-security-onion-proxmox-port-mirroring.37036/).
If you don't need to monitor traffic, you can install SO with only one interface.
Hope this helps.
